bukuserver/api.py

```diff
@@ -228,7 +228,7 @@ class ApiBookmarkSearchView(MethodView):
 
     def get(self):
         arg_obj = request.args
-        keywords = arg_obj.getlist('keywords')
+        keywords = arg_obj.getlist('keywords')  # pylint: disable=E1101
```

Comment:
Instead of disabling typecheck, shouldn't you clarify [the type of `request.args`](https://werkzeug.palletsprojects.com/en/2.2.x/datastructures/#werkzeug.datastructures.MultiDict)?

Reply:
@LeXofLeviafan what change is required here? I cna make the change and move forward. Please share the code.

Reply:
Instead of disabling type check, the correct way to fix the pylint error is to clarify the type of `request.args` (`# type: MultiDict`, probably along with an import from`werkzeug.datastructures`).
This applies to both instances of E1101.

Reply:
I am approving this change. Please raise the PR.